index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
```

Comment:
- [ ] [**OPTIONAL**] When the user clicks on the "Logo" or  "Portfolio" or "About" link, it does not take the user to the right section. It could better if you fix that. Make sure the IDs you used on the links matches the IDs of the sections.
It will make the user experience perfect :100: